moped-editor/src/components/GridTable/Search.js

```diff
@@ -184,6 +184,7 @@ const Search = ({
                 filtersConfig={filtersConfig}
                 resetSimpleSearch={resetSimpleSearch}
                 setIsOr={setIsOr}
+                searchTerm={searchTerm}
```

Comment:
I noticed that leading/trailing whitespace are preserved in the search term. Might be nice to trim that when the user executes the search?

Reply:
@tillyw could you sneak this change to trim in before merge? i noticed that the extra whitespace can make its way into the saved view descriptions as well 🙏

<img width="692" alt="Screenshot 2025-05-21 at 11 50 42 AM" src="https://github.com/user-attachments/assets/c69a0f77-ec27-4d3f-91d9-e794623b5cc8" />